pkg/vm/vm_test.go

```diff
@@ -869,6 +869,19 @@ func TestDivMod(t *testing.T) {
 		t.Run("negative/negative", runCase(opcode.MOD, -5, -2, -1))
 	})
 
+	t.Run("SHR", func(t *testing.T) {
```

Comment:
Well, I don't think it's a `DivMod` test after this.

Reply:
Fixed.